src/Packagist/WebBundle/Command/IndexPackagesCommand.php

```diff
@@ -12,12 +12,16 @@
 
 namespace Packagist\WebBundle\Command;
 
+use DateTime;
+use Exception;
```

Comment:
this goes against the Symfony coding standards, which are used in the Composer projects. It should be reverted